### ISSUE TITLE
Find the best way to implement rclone to HD2

### ISSUE THREAD

**Issue by kekesimarti** [filetrust/cdr-plugin-folder-to-folder#447]
We need to run multiple workflow clusters in parallel.
If you sync HD2 to a third-party engine, even if it's Minio or something else, you can start consuming directly from there.
The challenge is how to copy the data to multiple places.

**Comment by ggrig** [comment]
One of the possible ways would be to group files to N locations during preprocessing.
Then N workflow deployments will be able to process those in parallel where each deployment will pick up a single location

**Comment by ggrig** [comment]
s3fs (https://github.com/s3fs-fuse/s3fs-fuse) allows mounting both AWS S3 and non-Amazon S3 buckets (https://github.com/s3fs-fuse/s3fs-fuse/wiki/Non-Amazon-S3)
This works with the F2F plugin
However, for some reason the speed of communicating to Minio deployed in the local subnet is way too slow

**Comment by ggrig** [comment]
The solution has been implemented in https://github.com/filetrust/cdr-plugin-folder-to-folder/pull/462
The merge has been postponed as the solution needs running cdr-plugin container in privileged mode

**Comment by GINAGC** [comment]
@ggrig activity paused for now.
